bosh-release/jobs/wso2is/monit

```diff
@@ -1,7 +0,0 @@
-check process wso2is
```

Comment:
What would the reason for removing this file?

Reply:
@imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

Reply:
The PR shows some files have been moved but some as deleted and added as new.